packages/i18n-utils/src/localize-url.tsx

```diff
@@ -156,7 +156,8 @@ const urlLocalizationMapping: UrlLocalizationMapping = {
 export function localizeUrl(
 	fullUrl: string,
 	locale: Locale = getDefaultLocale(),
-	isLoggedIn = true
+	isLoggedIn = true,
+	skipHostLookup = false
```

Comment:
Could we add tests for this? I'm not sure what skipping the host lookup does.

Reply:
I pushed a few tests. Basically, this parameter prevents host-only path matches such as [wordpress.com](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-utils/src/localize-url.tsx#L134) because these allow any path to be localized (even for logged-in cases).